datalad/interface/common_cfg.py

```diff
@@ -460,7 +460,15 @@ def get_default_ssh():
     },
     'datalad.log.traceback': {
         'ui': ('question', {
-               'title': 'Runs TraceBack function with collide set to True, if this flag is set to "collide". This replaces any common prefix between current traceback log and previous invocation with "..."'}),
+               'title': 'Includes a compact traceback in a log message, with '
+                        'generic components removed. '
+                        'This setting is only in effect when given as an '
+                        'environment variable DATALAD_LOG_TRACEBACK. '
```

Comment:
do you give me permission to push to your PR adopted version of 
https://github.com/datalad/datalad/pull/6748/commits/0b604b6fccafe169b20ea9712cf29ec845aaa2b9
and adjust doc here as 
```suggestion
                        'This setting is in effect if set from command line via `-c` option. '
```

Reply:
I have not looked into that other commit closely. If you think this PR requires it, go ahead. Otherwise, I think a new PRs is more suitable than piling on.